pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.hyades</groupId>
   <artifactId>hyades</artifactId>
-  <version>0.2.0-SNAPSHOT</version>
+  <version>0.2.1-SNAPSHOT</version>
```

Comment:
Apologies, I gave the wrong answer earlier. This should be `0.1.3-SNAPSHOT`, the current latest version is `0.1.2`.

Reply:
The Maven release plugin automatically bumps the version to the next minor version, after the release is executed. Thus it ended up at `0.2.0-SNAPSHOT` after releasing `0.1.2`.